Runner.php

```diff
@@ -63,6 +65,20 @@ public function queuePaths($dir, $prefix = '', $exclusions = []) {
         }
     }
 
+    public function prefixCodeDirectory($configRulesets) {
+        $officialPhpRulesets = explode(',', Runner::RULESETS);
+        $configRulesets = explode(',', $configRulesets);
+
+        foreach ($configRulesets as &$r) {
+            if (!in_array($r, $officialPhpRulesets)) {
+             $r  = "/code/$r";
+             continue;
```

Comment:
Is this actually needed? There's nothing else in the loop after this, so I don't think it accomplishes anything.